src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -30,24 +30,27 @@
         on-import (:on-import props)]
     [comps/Button
      {:text "Import"
-      :onClick #(let [dest-conf-name (-> (@refs "destinationName") .getDOMNode .-value)
-                      dest-conf-namespace (-> (@refs "destinationNamespace") .getDOMNode .-value)]
-                 (endpoints/call-ajax-orch
-                   {:endpoint (endpoints/copy-method-config-to-workspace workspace-id)
-                    :payload {"configurationNamespace" selected-conf-namespace
-                              "configurationName" selected-conf-name
-                              "configurationSnapshotId" (str selected-conf-snapshot-id)
-                              "destinationNamespace" dest-conf-namespace
-                              "destinationName" dest-conf-name}
-                    :on-done (fn [{:keys [success? xhr]}]
-                               (if success?
-                                 (on-import {"name" dest-conf-name "namespace" dest-conf-namespace})
-                                 (js/alert (str "Import Error: " (.-responseText xhr)))))}))}]))
+      :onClick (fn []
+                 (let [dest-conf-name (-> (@refs "destinationName") .getDOMNode .-value)
+                       dest-conf-namespace (-> (@refs "destinationNamespace") .getDOMNode .-value)]
+                   ((fn [] (swap! state assoc :importing? true)))
```

Comment:
to achieve the desired function with the passed-in state.
Would it be preferred to use a  let binding here?